FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/logging-stack/logging_levels.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * FreeRTOS Kernel V10.3.0
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
```

Comment:
As the logging files are not meant to be distributed as a library, they should go under `/Demos` folder if it will be created for the `FreeRTOS-IoT-Libraries-LTS-Beta2` directory

Reply:
Are you saying that folder logging-stack is not good ? Just keep the files as is in /Demo/FreeRTOS-IoT-Libraries-LTS-Beta2? How about moving those to a /Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/common/ folder instead?